sdk/identity/ci.yml

```diff
@@ -30,6 +30,56 @@ extends:
   template: ../../eng/pipelines/templates/stages/archetype-sdk-client.yml
   parameters:
     ServiceDirectory: identity
+    TestMatrix:
+      # Default matrix
```

Comment:
Lets avoid duplicating this default matrix. Instead we should allow for adding additional test matrix entries.

Reply:
Sure @weshaggard . I will check it again and make required changes.